cypress/e2e/export-manager/search-filter-verify.cy.js

```diff
@@ -63,9 +68,19 @@ describe('export manager', () => {
 
         // Login and visit are separated, because otherwise user wasn't getting assigned permissions
         cy.login(userData.username, userData.password);
-        cy.visit(TopMenu.circulationLogPath);
+        cy.visit(TopMenu.bulkEditPath);
       });
 
+    // Creating a bulk edit job
+    FileManager.createFile(`cypress/fixtures/${userUUIDsFileName}`, userData.userId);
+    BulkEditSearchPane.checkUsersRadio();
+    BulkEditSearchPane.selectRecordIdentifier('User UUIDs');
+    BulkEditSearchPane.uploadFile(userUUIDsFileName);
+    BulkEditSearchPane.waitFileUploading();
+    BulkEditActions.downloadMatchedResults(matchRecordsFileName);
```

Comment:
Seems that is excess. I assume you can remove this line

Reply:
you're right, fixed